GeneralsMD/Code/Libraries/Source/WWVegas/WWAudio/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+set(WWAUDIO_SRC
+    "AudibleSound.cpp"
+    "AudioSaveLoad.cpp"
+    "FilteredSound.cpp"
+    "Listener.cpp"
+    "listenerhandle.cpp"
+    "LogicalListener.cpp"
+    "LogicalSound.cpp"
+    "sound2dhandle.cpp"
+    "Sound3D.cpp"
+    "sound3dhandle.cpp"
+    "SoundBuffer.cpp"
+    "soundhandle.cpp"
+    "SoundPseudo3D.cpp"
+    "SoundScene.cpp"
+    "SoundSceneObj.cpp"
+    "soundstreamhandle.cpp"
+    "Threads.cpp"
+    "Utils.cpp"
+    "WWAudio.cpp"
+)
+
+add_library(z_wwaudio STATIC)
+set_target_properties(z_wwaudio PROPERTIES OUTPUT_NAME wwaudio)
+
+target_sources(z_wwaudio PRIVATE ${WWAUDIO_SRC})
+
+target_link_libraries(z_wwaudio PRIVATE
+    gz_config
+    gz_utility
+    milesstub
+    winmm
+    z_ww3d2
+    z_wwdebug
+    z_wwmath
+    z_wwsaveload
+    zi_libraries_source_wwvegas_ww3d2
+    zi_libraries_source_wwvegas_wwdebug
+    zi_libraries_source_wwvegas_wwlib
+    zi_libraries_source_wwvegas_wwmath
+    zi_libraries_source_wwvegas_wwsaveload
```

Comment:
All these ww dependencies are required? I am asking because all the other ww libs have a lot less dependencies. Perhaps link to `wwcommon`?